config/clients/js/CHANGELOG.md.mustache

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.1.1
+
+### [0.1.1](https://{{gitHost}}/{{gitUserId}}/{{gitRepoId}}/compare/v0.1.0...v0.1.1) (2022-11-15)
+
+Updated [ListObjects API](https://openfga.dev/api/service#/Relationship%20Queries/ListObjects) which supports wildcard restriction in directly assignable type.
```

Comment:
I don't think this is the correct changelog - the wildcard is related to modeling, not ListObjects.

I would rename this:
- Regenerate to include support for restricting wildcards in authorization models [link to rfc]

Reply:
Updated.  Thanks!